go/enclave/rpc/rpc_encryption_manager.go

```diff
@@ -59,20 +61,20 @@ func (rpc *EncryptionManager) AddViewingKey(encryptedViewingKeyBytes []byte, sig
 	msgToSign := ViewingKeySignedMsgPrefix + hex.EncodeToString(viewingKeyBytes)
 
 	// We recover the key based on the signed message and the signature.
-	recoveredPublicKey, err := crypto.SigToPub(accounts.TextHash([]byte(msgToSign)), signature)
```

Comment:
I cleaned up the variable names here, as it was really unclear exactly what was being added to the map. I also added a comment on the field above.